src/Blocks/Zaken/Block.php

```diff
@@ -123,18 +123,22 @@ protected function getCombinedZaken(array $attributes): Collection
 
     protected function handleFilterOrdering(ZakenFilter $filter, array $attributes): ZakenFilter
     {
-        if (empty($attributes['orderBy'])) {
+        if (! isset($attributes['orderBy']) || ! is_string($attributes['orderBy'])) {
             return $filter;
         }
 
-        $filter->orderBy($attributes['orderBy']);
+        if (! isset($attributes['orderByDirection']) || ! is_string($attributes['orderByDirection'])) {
+            $attributes['orderByDirection'] = '-';
+        }
+
+        $filter->orderBy($attributes['orderBy'], $attributes['orderByDirection']);
```

Comment:
Moet je hier nog checken of orderByDirection wel een valide waarde is?

Reply:
Doe ik in de ZakenFilter class, dan hoeft de check in principe maar 1 keer plaats te vinden. Zelfde geldt voor de orderBy waarde.